src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -24,6 +24,11 @@ machine_type = "@MACHINETYPE@"
 # Default false
 # confidential_guest = true
 
+# Enable running QEMU VMM as a non-root user.
+# By default QEMU VMM run as root. When this is set to true, QEMU VMM process runs as
+# a non-root random user. See documentation for the limitations of this mode.
+# rootless = true
```

Comment:
do we need an annotation for this?  this way rootless can be enable or disable through k8s annotations, wdyt?

Reply:
I don't have a preference. Maybe we can defer this to a later time?

Reply:
ok, just file an issue please

Reply:
Updated.

Reply:
As I was reviewing your PR for adding documentation (https://github.com/kata-containers/kata-containers/pull/2546), I am finding this terminology a bit confusing. 
I would like to make sure we clearly make the distinction between true `rootless` mode where the runtime itself is invoked as rootless from the upper stack vs what is being done here i.e running the hypervisor as rootless.

I would like to propose we name this configuration something else to avoid confusion to other users, something like `rootless-hypervisor`. 
Would like to see what others think about this.
cc @jodh-intel @egernst @fidencio @bergwolf @c3d

Reply:
all options in this section (`[hypervisor]`)  **only** have effect on the hypervisor, so adding suffixes like `hypervisor` or `vmm`  are useless / pointless.